calculator_test.go

```diff
@@ -2,23 +2,310 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
 )
 
-func TestAdd(t *testing.T) {
+func TestAddSubtractMultiply(t *testing.T) {
 	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
-	if want != got {
+	testCases := []struct {
+		name       string
+		a, b, want float64
+		f          func(float64, float64, ...float64) float64
+	}{
+		{name: "Add 1 + 1", a: 1, b: 1, want: 2, f: calculator.Add},
```

Comment:
I suggest writing all of these as multiline literals (or, at least, consistently writing them as either single-line or multiline; don't mix the two).